### DISCUSSION TITLE
Engine do not open the project

### DISCUSSION THREAD

**Discussion by nathancmiguel** [o3de/o3de#12586] [Help Wanted]
I have a computer with a 750 ti. But when I compile the gems and try to open nothing happen, no error or warning. Is that my hardware?

**Comment by Nodoubvt** [comment]
Good luck getting any kinda of response from this community..   or devs.

But to answer your question, yes it's your hardware. Min. video card spec is 1060ti  I believe. Just remember anything you make with this engine will require your customers to meet these high demands as well, thus cutting your potential audience down by 80%+.  This is not the droids...  errr i mean this is not the engine your looking for.. move along.. move along.
